### PR TITLE
keep running tally of grants claimed on a per-ip basis

### PR DESCRIPTION
**Pull request by evq**

### CODE REVIEW

File: controllers/grants.go [hunk 1]
```diff
@@ -81,6 +84,21 @@ func ClaimGrant(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	conn := datastore.GetRedisConn(r.Context())
+
+	if err == nil {
```
Comment (jumde) [inline controllers/grants.go]: `err` here is the error returned from `govalidator.ValidateStruct`. Not sure if this is required.

Comment (evq) [reply controllers/grants.go]: good catch, yeah this is totally redundant - I will remove it

File: controllers/grants.go [hunk 2]
```diff
@@ -81,6 +84,21 @@ func ClaimGrant(w http.ResponseWriter, r *http.Request) {
...
+		// FIXME TODO clean this up via a better abstraction
+		if _, err := redis.Int((*conn).Do("ZINCRBY", "count:claimed:ip", "1", r.RemoteAddr)); err != nil {
+			raven.CaptureMessage("Could not increment claim count for ip.", map[string]string{"IP": r.RemoteAddr})
```
Comment (jumde) [inline controllers/grants.go]: Not sure how the db is designed, but may be it'll be good to capture the error string with the IP?

Comment (evq) [reply controllers/grants.go]: `CaptureMessage` takes a string message and an optional dictionary of additional metadata. Sentry treats each distinct string as a different event type, but does keep track of the additional metadata.

**Pull request merged**